src/flownet/ahm/_ahm_iteration_analytics.py

```diff
@@ -166,7 +165,7 @@ def accuracy_metric(data_reference, data_test, metric):
     if metric in ("MSE", "RMSE", "NRMSE"):
         score = mean_squared_error(data_reference, data_test)
         if metric in ("RMSE", "NRMSE"):
-            score = math.sqrt(score)
+            score = mean_squared_error(data_reference, data_test, squared=False)
```

Comment:
I'd suggest to make this function more readable by rewriting it to something like (lines 168-180):

```python
if metric is "MSE":
    score = mean_squared_error(data_reference, data_test)
elif metric is "RMSE":
    score = mean_squared_error(data_reference, data_test, squared=False)
elif metric is "NMRSE":
    score = mean_squared_error(data_reference, data_test, squared=False) / (np.amax(data_reference) - np.amin(data_reference))
elif metric is "MAE":
    score = mean_absolute_error(data_reference, data_test)
elif metric is "NMAE":
    score = mean_absolute_error(data_reference, data_test) / (np.amax(data_reference) - np.amin(data_reference))
elif metric is "R2":
    score = r2_score(data_reference, data_test, multioutput="variance_weighted")
else:
    raise NotImplementedError(f"The metric '{metric}' is not supported.")

return score
```